Draft mp41 certs (#99)
OpenLiberty/certifications: microprofile/4.1/TCKResults.adoc

:page-layout: certification
= MicroProfile 4.1 TCK Results Summary

As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the MicroProfile TCK results for releases of MicroProfile 4.1.

== Open Liberty 20.0.0.3 MicroProfile 4.1 Certification Summary

> MicroProfile 4.0 consists of 8 MicroProfile Specifications and 4 Jakarta EE 8 specfications.  

Reviewer: ```suggestion
MicroProfile 4.1 consists of 8 MicroProfile Specifications and 5 Jakarta EE 8 specfications.  
```

Author: Done

* MicroProfile Config 2.0

* MicroProfile Fault Tolerance 3.0

* MicroProfile Health 3.1

* MicroProfile JWT Authentication 1.2

* MicroProfile Metrics 3.0

* MicroProfile OpenAPI 2.0

* MicroProfile OpenTracing 2.0

* MicroProfile Rest Client 2.0

> * Jakarta Contexts and Dependency Injection 2.0

Reviewer: ```suggestion
* Jakarta Contexts and Dependency Injection 3.0
```

> * Jakarta RESTful Web Services 2.1

Reviewer: ```suggestion
* Jakarta RESTful Web Services 2.1
```
```suggestion
* Jakarta RESTful Web Services 3.0
```

> * Jakarta RESTful Web Services 2.1

Reviewer: ```suggestion
* Jakarta RESTful Web Services 3.0
```

> * Jakarta JSON Binding 1.0

Reviewer: ```suggestion
* Jakarta JSON Binding 2.0
```

> * Jakarta JSON Processing 1.1

Reviewer: ```suggestion
* Jakarta JSON Processing 2.0
```

> * Jakarta JSON Processing 1.1

Reviewer: Jakarta Annotations 1.3

Author: Done

> Below are the TCK summaries for each specification.

Reviewer: ```suggestion
According to the requirement of MicroProfile 4.1, in order to certify against MicroProfile 4.1. The included eight MicroProfile component specifications must pass the corresponding TCK tests, while passing the included give Jakarta EE 8 specifications are optional. Below are the TCK summaries for the eight MicroProfile specifications.
```

Author: Done

=== MicroProfile Component Sepecification Certification Summary
* link:https://openliberty.io/certifications/microprofile/4.1/config/2.0/TCKResults.html[MicroProfile Config 2.0 TCK Result]

* link:https://openliberty.io/certifications/microprofile/4.1/faulttolerance/3.0/TCKResults.html[MicroProfile Fault Tolerance 3.0 TCK Result]

* link:https://openliberty.io/certifications/microprofile/4.1/health/3.1/TCKResults.html[MicroProfile Health 3.1 TCK Result]

* link:https://openliberty.io/certifications/microprofile/4.1/mpjwt/1.2/TCKResults.html[MicroProfile JWT Authentication 1.2 TCK Result]

* link:https://openliberty.io/certifications/microprofile/4.1/metrics/3.0/TCKResults.html[MicroProfile Metrics 3.0 TCK Result]

* link:https://openliberty.io/certifications/microprofile/4.1/openapi/2.0/TCKResults.html[MicroProfile OpenAPI 2.0 TCK Result]

* link:https://openliberty.io/certifications/microprofile/4.1/opentracing/2.0/TCKResults.html[MicroProfile OpenTracing 2.0 TCK Result]

* link:https://openliberty.io/certifications/microprofile/4.1/restclient/2.0/TCKResults.html[MicroProfile Rest Client 2.0 TCK Result]

> === Four Jakarta EE 8 Sepecification Certification Summary

Reviewer: ```suggestion
=== Optional: Five Jakarta EE 8 Sepecification Certification Summary
```

Author: Done

> This link:https://openliberty.io/certifications/jakartaee/webprofile/8/20.0.0.3-TCKResults.html[Jakarta EE 8 TCK Summary] lists the TCK Results for the following 4 Jakarta EE 8 specifications.

Reviewer: ```suggestion
This link:https://openliberty.io/certifications/jakartaee/webprofile/8/20.0.0.3-TCKResults.html[Jakarta EE 8 TCK Summary] lists the TCK Results for the following 5 Jakarta EE 8 specifications.
```
```suggestion
This link:https://openliberty.io/certifications/jakartaee/webprofile/8/20.0.0.3-TCKResults.html[Jakarta EE 8 TCK Summary] lists the TCK Results for the following 4 Jakarta EE 8 specifications.
```

Author: Done
